README.md

```diff
@@ -1,2 +1,161 @@
+The Blog app will be a classic example of a blog website. You will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
+
+<a name="readme-top"></a>
+
+<h3><b>Blog_App</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+   - [🛠 Built With](#built-with)
+      - [Tech Stack](#tech-stack)
+      - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+   - [Prerequisites](#prerequisites)
+   - [Setup](#setup)
+   - [Install](#install)
+   - [Usage](#usage)
+   - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Blog_App<a name="about-project"></a>
+
+**Blog_App** is a fully functional web application that showcases the classic example of a blog website. It allows users to explore a list of posts and actively engage with the content by adding comments and liking posts. This project demonstrates the power of Ruby on Rails in creating dynamic and interactive web experiences.
+
+## 🛠 Built With <a name="built-with">Ruby on Rails</a>
+
+### Tech Stack <a name="tech-stack">Rspec</a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Integration specs for Views**
+- **Validations, Model specs, and n+1 problems**
+- **Add authorization rules**
+- **Add API endpoints**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- `Ruby 3.2.2`
+- `Rails`
+- `VSCode` or `RubyMine`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:Enning94/Blog_App.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Blog_app
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  ruby result.rb
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  bundle exec rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Enning Festus**
+
+- GitHub: [@Enning94](https://github.com/Enning94)
+- Twitter: [@nana_akyerefi](https://twitter.com/nana_akyerefi)
+- LinkedIn: [Enning Festus](https://www.linkedin.com/in/enningfestus/)
+
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
```

Comment:
- [ ] Kindly include at least `2 Future Features` in your README file instead of leaving it blank to make your README more professional and also because it is a required section in the MIcroverse README template :+1: